operators/multiclusterobservability/api/shared/multiclusterobservability_shared.go

```diff
@@ -39,6 +39,12 @@ type PreConfiguredStorage struct {
 	// Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
 	// +required
 	Name string `json:"name"`
+	// TLS secret contains the custom certificate for the object store
+	// +optional
+	TLSSecretName string `json:"tlsSecretName,omitempty"`
+	// TLS secret mount path in thanos store/ruler/compact/receiver
+	// +optional
+	TLSSecretMountPath string `json:"tlsSecretMountPath,omitempty"`
```

Comment:
is it necessary to expose "mountpath" to end user?

Reply:
Yes. I think it is necessary to expose "mountpath" to end user, because you have to configure the full path of certs in objectstore. it is directly using by thanos components. for example: `ca_file: /etc/minio/certs/ca.crt`. the store component find this file and use this ca.crt. if user does not support `mountpath`, we cannot mount the cert secret into the correct path. Does it make sense?

Reply:
Parse the `ca_file` to get the mountpath from object storage secret. Thanks @marcolan018 for your comments.